CONFIGURE.md

```diff
@@ -0,0 +1,405 @@
+
+
+    Flow Controller:
+        id: 471deef6-2a6e-4a7d-912a-81cc17e3a205
+        name: MiNiFi Flow
+
+    Processors:
+        - name: GetFile
+          id: 471deef6-2a6e-4a7d-912a-81cc17e3a206
+          class: org.apache.nifi.processors.standard.GetFile
+          max concurrent tasks: 1
+          scheduling strategy: TIMER_DRIVEN
+          scheduling period: 1 sec
+          penalization period: 30 sec
+          yield period: 1 sec
+          run duration nanos: 0
+          auto-terminated relationships list:
+          Properties:
+              Input Directory: /tmp/getfile
+              Keep Source File: true
+
+    Connections:
+        - name: TransferFilesToRPG
+          id: 471deef6-2a6e-4a7d-912a-81cc17e3a207
+          source name: GetFile
+          source id: 471deef6-2a6e-4a7d-912a-81cc17e3a206
+          source relationship name: success
+          destination id: 471deef6-2a6e-4a7d-912a-81cc17e3a204
+          max work queue size: 0
+          max work queue data size: 1 MB
+          flowfile expiration: 60 sec
+
+    Remote Processing Groups:
+        - name: NiFi Flow
+          id: 471deef6-2a6e-4a7d-912a-81cc17e3a208
+          url: http://localhost:8080/nifi
+          timeout: 30 secs
+          yield period: 10 sec
+          Input Ports:
+              - id: 471deef6-2a6e-4a7d-912a-81cc17e3a204
+                name: From Node A
+                max concurrent tasks: 1
+                Properties:
+
+### SiteToSite Security Configuration
+
+    in minifi.properties
+
+    enable tls
+    nifi.remote.input.secure=true
+
+    if you want to enable client certificate base authorization
+    nifi.security.need.ClientAuth=true
+    setup the client certificate and private key PEM files
+    nifi.security.client.certificate=./conf/client.pem
+    nifi.security.client.private.key=./conf/client.pem
+    setup the client private key passphrase file
+    nifi.security.client.pass.phrase=./conf/password
+    setup the client CA certificate file
+    nifi.security.client.ca.certificate=./conf/nifi-cert.pem
+
+    if you do not want to enable client certificate base authorization
+    nifi.security.need.ClientAuth=false
+  
+You have the option of specifying an SSL Context Service definition for the RPGs instead of the properties above. 
+This will link to a corresponding SSL Context service defined in the flow. 
+    
+To do this specify the SSL Context Service Property in your RPGs and link it to 
+a defined controller service. If you do not take this approach the options, above, will be used
+for TCP and secure HTTPS communications.
+    
+    Remote Processing Groups:
+    - name: NiFi Flow
+      id: 2438e3c8-015a-1000-79ca-83af40ec1998
+      url: http://127.0.0.1:8080/nifi
+      timeout: 30 secs
+      yield period: 5 sec
+      Input Ports:
+          - id: 2438e3c8-015a-1000-79ca-83af40ec1999
+            name: fromnifi
+            max concurrent tasks: 1
+            Properties:
+                Port: 10443
```

Comment:
Don't need Port and Host Name for our RPG ports unless I'm missing a condition of using the context service